docs/providers/aws/events/event-bridge.md

```diff
@@ -51,9 +53,9 @@ functions:
 
 The `eventBridge` event source will use the `default` event bus (the one AWS uses internally) when none is explicitly specified.
 
-The Serverless Framework will create the `eventBus` for you if you just provide a name for it. It will re-use an existing event bus if you provide an event bus `arn`.
+The Serverless Framework will create the `eventBus` for you if you just provide a name for it. It will re-use an existing event bus if you provide an event bus `arn`. Additionally, if you provide event bus with CloudFormation intrinsic function, it also won't be created.
 
-**NOTE:** The Serverless Framework won't manage (e.g. create or remove) the event bus if it's provided via an `arn`.
+**NOTE:** The Serverless Framework won't manage (e.g. create or remove) the event bus if it's provided via an `arn` or is configured with CloudFormation intrinsic function.
```

Comment:
I think this additional note is needed. It feels natural (goes without saying), that if we attach to _existing_ event bus, we won't attempt to create or manage it